tools/hidl/hidlcomp.cpp

```diff
@@ -4995,11 +3951,8 @@ void EspServInfo::write_esp_binding_ipp()
     //method ==> processRequest
     outs("\tvirtual int processRequest(IRpcMessage* rpc_call, IRpcMessage* rpc_response);\n");
 
-    //method ==> getXsdDefinition
-    outs("\tint getXsdDefinition(IEspContext &context, CHttpRequest* request, StringBuffer &content, const char *service, const char *method, bool mda);\n");
-
-    //method ==> getMethodHtmlForm
-    outs("\tvirtual int getMethodHtmlForm(IEspContext &context, CHttpRequest* request, const char *serv, const char *method, StringBuffer &page, bool bIncludeFormTag);\n");
+    // method ===> getXmlFilename
```

Comment:
Correct comment to read getServiceXmlFilename

Reply:
Updated.